client/vite.config.js

```diff
@@ -25,5 +25,8 @@ export default defineConfig({
         },
       },
     },
+    watch: {
+      usePolling: true,
```

Comment:
Turning on polling is not great for performance- are you running everything in Docker? On what host OS?

Reply:
I was using usePolling because Docker wouldnt catch the changes I made to the code. I'm using Windows. I'll take it out! Is there a practical way for docker to detect file changes?